template/python3-http-armhf/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM openfaas/of-watchdog:0.5.3 as watchdog
+FROM openfaas/of-watchdog:0.5.4-armhf as watchdog
```

Comment:
This should not be present `-armhf` - it's a multi-arch build, so we can use the "generic" multi-arch name

Reply:
Fixed removed -armhf